### DISCUSSION TITLE
最新版代码里面，3rd/lua里面的luac.c到哪里去了?

### DISCUSSION THREAD

**Discussion by liyqxtu** [cloudwu/skynet#1736] [General]
gcc -std=gnu99 -o lua.exe -s  lua.o lua54.dll -lm
make[2]: Leaving directory '/d/skynet-mingw/3rd/lua'
make "LUAC_T=luac.exe" luac.exe
make[2]: Entering directory '/d/skynet-mingw/3rd/lua'
make[2]: *** No rule to make target 'luac.c', needed by 'luac.o'.  Stop.
make[2]: Leaving directory '/d/skynet-mingw/3rd/lua'
make[1]: *** [makefile:133: mingw] Error 2
make[1]: Leaving directory '/d/skynet-mingw/3rd/lua'
make: *** [Makefile:79: 3rd/lua/liblua.a] Error 2

查了一下代码，确实是没有3rd/lua/luac.c，但是makefile里面有，所以导致了该编译错误

**Comment by cloudwu** [comment]
lua 官方仓库把 luac 另列了。

skynet 并不依赖 luac 。
